performance/main.py

```diff
@@ -32,6 +38,41 @@ class ImmutableFilterAK(ImmutableFilter):
 class ImmutableFilterREF(ImmutableFilter):
     entry = staticmethod(immutable_filter_ref)
 
+#-------------------------------------------------------------------------------
+class ShapeFilter(Perf):
+
+    def pre(self):
+        self.array1 = np.arange(100)
+        self.array2 = self.array1.reshape(20, 5)
+
+    def main(self):
+        _ = self.entry(self.array1)
+        _ = self.entry(self.array2)
```

Comment:
I assume you're doing these weird assignments for Pylint?

Perhaps we can clean this by just disabling that error by putting `pylint: disable = whatever-error-name` at the top of this file.

Reply:
nah, not pylint yet, but just not sure if it is useful to show that these functions return something. I am fine to remove them!